.github/workflows/check.yml

```
@@ -34,6 +34,18 @@ jobs:
         with:
           persist-credentials: false
 
+      # TODO: Lock this Action to a release tag once commit [1] ("fix: relocate
+      # TMPDIR to /mnt to improve Nix installer compatibility") is part of a
+      # stable release, resolving [2] ("does not work with cachix install nix
+      # action").
+      #
+      # [1]: https://github.com/wimpysworld/nothing-but-nix/pull/25
+      # [2]: https://github.com/wimpysworld/nothing-but-nix/issues/24
+      - uses: wimpysworld/nothing-but-nix@main
```

we can also pin to a commit
```suggestion
      - uses: wimpysworld/nothing-but-nix@7024aede12666f4c03704df6ae740a4adf9b6682
```